todo-app/src/App.js

```diff
@@ -1,13 +1,23 @@
 import React, { Component } from 'react';
+import TodosList from './todosList'
+import Todo from './todo'
+import './App.css'
 
 class App extends Component {
   render() {
     return (
-      <div>
-        Hello World!
+      <div className='main'>
+        <h1>Todo List App</h1>
+        {TodosList.map((todo, index) => {
+          return (
+          <Todo
+          todoDescription={todo.todoDescription}
```

Comment:
You could pass the `todo` as a whole so you don't need to extend the props if `todo` gets more properties that we want to render inside `Todo`. So something like

```jsx
<Todo todo={todo} />
```